src/statmodels.jl

```diff
@@ -336,12 +336,31 @@ function show(io::IO, ct::CoefTable)
 end
 
 """
-	ConvergenceException(iters::Int)
+	ConvergenceException(iters::Int, change::Real=NaN, tol::Real=Nan)
 
 The fitting procedure failed to converge in `iters` number of iterations.
+i.e. where the `change` between the cost of the final & penultimate iteration was greater than
```

Comment:
Comma rather than dot in the previous sentence. "and" rather than "&". Remove "where".

Also, calling the field/argument `lastchange` could be more explicit?